src/Generics/Linear/Unsafe/ViaGHCGenerics.hs

```diff
@@ -97,84 +107,75 @@ instance G.Generic a => Generic (GHCGenerically a) where
 -- completely bogus results, breaking the linearity rules.
 --
 -- @GHCGenerically1@ is otherwise safe to use with /derived/
--- @"GHC.Generics".'G.Generic1'@ instances, which are linear. If
+-- @"GHC.Generics".'G.Generic'@ instances, which are linear. If
 -- you choose to use it with a hand-written instance, you should
 -- check that the underlying instance is linear.
 --
 -- === Example
 --
 -- @
 -- data Foo a = Bar a (Either Int a) | Baz (Maybe a) Int
---   deriving stock (Show, "GHC.Generics".'G.Generic1')
+--   deriving stock (Show, "GHC.Generics".'G.Generic')
 --   deriving 'Generic1' via GHCGenerically1 Foo
 -- @
 type GHCGenerically1 :: forall k. (k -> Type) -> k -> Type
 newtype GHCGenerically1 f a = GHCGenerically1 { unGHCGenerically1 :: f a }
 
-instance (G.Generic1 f, Repairable ('ShowType f) (G.Rep1 f)) => Generic1 (GHCGenerically1 f) where
-  type Rep1 (GHCGenerically1 f) = Repair (G.Rep1 f)
-
-  -- Why do we use 'unsafeCoerce' for these rather than just 'toLinear'?
```

Comment:
It's a shame that this PR removes these comments about `unsafeCoerce`. Can you restore them next to the new call sites to `unsafeCoerce` in the `GenericQ f a` instance?

Reply:
I think the comments were a bit wrong, but I'll add a fixed and updated version, yeah.